### ISSUE TITLE
Crackling Audio with VoiceMeeter ASIO

### ISSUE THREAD

**Issue by 27siya** [Andersama/obs-asio#97]
The plugin receives the audio with severe crackling, when connecting to VoiceMeeter Asio, with the audio itself being just fine.

**Comment by 27siya** [comment]
Just want to mention that the Insert ASIO is working fine, audio sounds perfect, but the others don't. Just find that weird, but for me the issue is resolved.

**Comment by Andersama** [comment]
You're not the only one: https://github.com/Andersama/obs-asio/pull/90#issuecomment-855311393

Voicemeeter at least for me is unstable and becomes crackly with other applications in a similar fashion. I did write a plugin specifically for it if you'd like to try that out.

**Comment by 27siya** [comment]
Yes please ! I would very much appreciate it. Brilliant work by the way.
